src/main/java/com/booster/server/VocabularyEntryService.java

```diff
@@ -16,9 +19,24 @@ public VocabularyEntryDto create(CreateVocabularyEntryInput input) {
 
         vocabularyEntryEntity = vocabularyEntryRepository.save(vocabularyEntryEntity);
 
+        return getVocabularyEntryDto(vocabularyEntryEntity);
+    }
+
+    public VocabularyEntryListDto list(Integer size) {
+        List<VocabularyEntryDto> list = vocabularyEntryRepository.findAll()
```

Comment:
Querying for `all` entities is a bad practice. Please implement a custom `JpaRepository` method to return a limited response